src/app/components/service-down/service-down.component.html

```diff
@@ -3,7 +3,9 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">{{currentError.header}}</h1>
       <p class="govuk-body" *ngFor="let error of currentError.errors">
-        <a *ngIf="error.urlText" routerLink="{{error.url}}">{{error.urlText}}</a> {{ error.bodyText }}
+        <a *ngIf="error.urlText && !error.newTab" routerLink="{{error.url}}">{{error.urlText}}</a> 
+        <a *ngIf="error.urlText && error.newTab" routerLink="{{error.url}}" target="_blank">{{error.urlText}}</a> 
```

Comment:
I was once told that having "&&" in conditions inside templates is not best practise, as the cases you need to cover increases (in this example it is "!error.urlText && !error.newTab" and "!error.urlText && error.newTab"). You could maybe build a separate condition for target. true=_blank, false=_self

Reply:
@adnanakgun  I have removed the && now and added functions for it. This should address your concern. Also added tests for the function. Thanks.